pkg/controller/import-controller.go

```diff
@@ -23,6 +23,8 @@ const (
 	AnnImportPod = "cdi.kubevirt.io/storage.import.importPodName"
 	//LabelImportPvc is a pod label used to find the import pod that was created by the relevant PVC
 	LabelImportPvc = "cdi.kubevirt.io/storage.import.importPvcName"
+	//AnnCreateImage provides a const for our PVC createImage annotation
+	AnnCreateImage = "cdi.kubevirt.io/storage.import.createImage"
```

Comment:
Again, not sure `create` is the right word.  Because an image always gets created.  What makes this special is that it's blank/empty.  I may be off here, but what do others think?

Reply:
until we get another decision, I changed to 'blank'